modules/selenium/src/test/java/org/testcontainers/junit/SpecificImageNameWebDriverContainerTest.java

```diff
@@ -8,7 +8,9 @@
 
 public class SpecificImageNameWebDriverContainerTest extends BaseWebDriverContainerTest {
 
-    private static final DockerImageName FIREFOX_IMAGE = DockerImageName.parse("selenium/standalone-firefox:2.53.1-beryllium");
+    private static final DockerImageName FIREFOX_IMAGE = DockerImageName
+        .parse("selenium/standalone-firefox:2.53.1-beryllium")
+        .asCompatibleSubstituteFor("selenium/standalone-firefox-debug");
```

Comment:
the same as with Elasticsearch - we may need to be able to verify the compatibility with multiple images

Reply:
Yep, have replaced this by something more elegant within `BrowserWebDriverContainer`